### PR TITLE
Noticket tweaks

### PR DESCRIPTION
**Pull request by benji6**
Just some ideas after reading through the codebase - hopefully some of them useful! :D

### CODE REVIEW

**Comment by benji6** [comment]
I wasn't really sure whether we can easily share the browserlist between cssnext and babel - ideas?

**Comment by HHogg** [comment]
@benji6 You `.babelrc` can be a JS file :) _(.babelrc.js)_

File: .babelrc [hunk 1]
```diff
@@ -1,8 +1,22 @@
 {
+  "plugins": [
+    "transform-class-properties",
+    "transform-object-rest-spread"
+  ],
   "presets": [
-    ["es2015", { "modules": false }],
     "react",
-    "stage-2"
+    ["env", {
```
Comment (HHogg) [inline .babelrc]: Double check HMR, I think @thatguynamedandy tried swapping over to this and ran into some issues.

Comment (thatguynamedandy) [reply .babelrc]: I'll maybe checkout the branch and see if it all still works, I'm in inclined to think I had some kind of caching issue when I tried it, there's no reason it shouldn't work.

**Comment by benji6** [comment]
@HHogg as we've realised you can only do `.babelrc.js` in the next version of babel! Shall we leave it like this until our next upgrade :D

File: .babelrc (hunk 1 above)
Comment (thatguynamedandy) [reply .babelrc]: I double checked, It seems to work now, I must of had a local machine issue.

**Comment by thatguynamedandy** [comment]
I'm happy to leave it like that, it would be great if we could get something like https://greenkeeper.io/ in so that we keep up to date with dependencies.

**Pull request merged**